documentation/Installation.md

```diff
@@ -577,21 +579,23 @@ node:
   - name: "lb"
     username: "centos"
     keyfile: "/home/username/.ssh/id_rsa"
+    password: "your_password"
     internal_address: "192.168.0.1"
     roles: ["balancer"]
   - name: "control-plane"
     username: "centos"
     keyfile: "/home/username/another.key"
+    password: "your_password"
     internal_address: "192.168.0.2"
     roles: ["control-plane"]
 ```
 
 Following are the parameters allowed to be specified in the `node_defaults` section:
-* keyfile, username, connection_port, connection_timeout and gateway.
+* keyfile, password, username, connection_port, connection_timeout and gateway.
 * labels, and taints - specify at global level only if the [Mini-HA Scheme](#mini-ha-scheme) is used.
-
 For more information about the listed parameters, refer to the following section.
 
+Note: To establish an ssh connection, you can use either a keyfile or a password.
```

Comment:
Please specify more info, about `keyfile` and `password` fields: different configurations, what happens, if user specifies both of them, the limitations, that @ilia1243 discussed, and so on.